package.json

```diff
@@ -52,29 +52,26 @@
     "@babel/plugin-proposal-class-properties": "^7.8.3",
     "@babel/preset-env": "^7.9.5",
     "@babel/preset-react": "^7.9.4",
+    "@types/react": "^16.14.32",
     "babel-eslint": "^10.1.0",
-    "babel-jest": "^25.4.0",
+    "babel-jest": "^29.2.1",
     "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.23.0",
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
-    "eslint": "^6.8.0",
-    "eslint-config-airbnb": "^18.1.0",
-    "eslint-config-prettier": "^6.10.1",
+    "eslint": "^8.25.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-import": "^2.20.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.19.0",
-    "jest": "^25.4.0",
```

Comment:
I wonder if we should leave the unit test package version as is and update them as we unit tests just in case they cause more headaches

Reply:
Yeah -- this was kind of a mess because the existing enzyme version didn't work very well with updated React/Typescript versions. 

Ended up doing a bunch of updates and disabling a handful (16) of tests -- will create issues for each of those and tackle them individually so I can get this merged in.